<launch>
	<node name="front_lt_camera_vew" pkg="image_vew" type="image_view" />
	<node name="front_lt_camera_vew" pkg="image_vew" type="image_view" />
	<node name="front_lt_camera_vew" pkg="image_vew" type="image_view" />
	<node name="front_lt_camera_vew" pkg="image_vew" type="image_view" />
	
	<node pkg="
</launch>
